src/mocks/setupMocks.ts

```diff
@@ -86,5 +86,18 @@ export const worker = setupWorker(
     return new HttpResponse(null, {
       status: 200
     })
+  }),
+  http.post('/mock/pamelding/:deltakerId/kladd', async ({request, params}) => {
+    await delay(1000)
+    const { deltakerId } = params
+    const requestBody = await request.json()
+
+    // eslint-disable-next-line no-console
+    console.log('Kladd stored', deltakerId, requestBody)
```

Comment:
Skal vi ha med den?

Reply:
Gjør ikke så mye siden det er i mocken da

Reply:
Tenker det kan være fint å se hva som kommer gjennom i mocken, men kan godt fjerne den om det blir for mye mas?